src/java.base/share/classes/java/lang/classfile/Attribute.java

```diff
@@ -66,7 +66,7 @@
 import jdk.internal.javac.PreviewFeature;
 
 /**
- * Models a classfile attribute {@jvms 4.7}.  Many, though not all, subtypes of
+ * Models a classfile attribute JVMS {@jvms 4.7}.  Many, though not all, subtypes of
```

Comment:
```suggestion
 * Models a classfile attribute (JVMS {@jvms 4.7}).  Many, though not all, subtypes of
```
Same as for all other attributes.